### PR TITLE
MBS-6514 Remove android-maven-publish

### PR DESCRIPTION
**Pull request by sergioboi**
publish via agp https://developer.android.com/studio/build/maven-publish-plugin

### CODE REVIEW

**Comment by dsvoronin** [comment]
Wow! Thanks for that.

How did you test it?

**Review by dsvoronin — APPROVED** [review]

File: build.gradle.kts [hunk 1]
```diff
@@ -138,11 +138,19 @@ subprojects {
             }
         }
     }
-
-    plugins.withType<LibraryPlugin> {
-        tasks.register<Jar>(sourcesTaskName).configure {
-            archiveClassifier.set("sources")
-            from(this@withType.extension.sourceSets["main"].java.srcDirs)
+    plugins.withId("com.android.library") {
```
Comment (dsvoronin) [inline build.gradle.kts:141]: Is type -> string weakening necessary here?

Comment (sergioboi) [reply build.gradle.kts:141]: No, we can replace by Type

**Comment by sergioboi** [comment]
> Wow! Thanks for that.
> 
> How did you test it?

I test it locally. I haven't tested a bintray scenario only. Because I didn't get how.

**Pull request merged**